sdk/tables/azure-data-tables/azure/data/tables/_table_batch.py

```diff
@@ -40,6 +40,18 @@ class TableBatchOperations(object):
     supported within a single transaction. The batch can include at most 100
     entities, and its total payload may be no more than 4 MB in size.
 
+    :param client: An AzureTable object.
+    :type client: ~azure.data.tables._generated.AzureTable
+    :param serializer: A Serializer object for request serialization.
+    :type serializer: ~azure.data.tables._generated._serialization.Serializer
+    :param deserializer: A Deserializer object for request deserialization.
+    :type deserializer: ~azure.data.tables._generated._serialization.Deserializer
+    :param config: An AzureTableConfiguration object.
+    :type config: ~azure.data.tables._generated._configuration.AzureTableConfiguration
```

Comment:
I don't think you want to expose private models in documentation. Is this necessary?

Reply:
This class is for internal use, so I think it's ok.